### PR TITLE
is-powerless() tests

### PR DESCRIPTION
**Pull request by oluoluoxenfree**

### CODE REVIEW

File: spec/core_functions/color/is_powerless.hrx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<===> no_space/input.scss
+a {b: is-powerless('red', 'hue')}
+
+<===> no_space/output.scss
```
Comment (dvdherron) [inline spec/core_functions/color/is_powerless.hrx:4]:
```suggestion
<===> no_space/output.css
```
Here and throughout this file, the output files should end with `.css`

**Review by dvdherron — COMMENTED** [review]
From just giving this a quick glance:
 - I compare the test/expectation format with some of the other related tests. The output files need to be `.css` or else the test throws an errror. 
 -  the spec says the channel should be an unquoted string. I think they are all currently quoted strings. 
 - this function also has an optional `$space` argument. We probably want to test this function with some different spaces.
 - Have you tried running the tests locally? if you run `npm run sass-spec -- --interactive --dart $DART_SASS_PATH spec/core_functions/color/is_powerless.hrx` you should be able to see which tests are failing and why. (To get the latest test results, your clone of `dart-sass` should be on the `feature.color-4` branch.)
 Edit: - I would also test channels for more of the newer color space channels (display-p3, lab/oklab, etc.)
- and see some of the other tests that are generally included in all tests (e.g: there is probably always a test for named arguments)

**Comment by oluoluoxenfree** [comment]
@dvdherron I haven't run them locally I didn't know what was an option without pulling in all the other stuff, will do! 

thank you for the other pointers also!

File: spec/core_functions/color/is_powerless.hrx (hunk 1 above)
Comment (oluoluoxenfree) [reply spec/core_functions/color/is_powerless.hrx:4]: @dvdherron should all of the files end with css or is it just this one? Why?

Comment (dvdherron) [reply spec/core_functions/color/is_powerless.hrx:4]: @oluoluoxenfree All of the output files should end with `.css`. The tests are taking the input, compiling it to CSS and comparing the results. If the the output file doesn't end in `.css` (error files follow a different format) the test runner will throw an error and say that the `output` file is missing.

**Pull request closed without merging**